react-exercise/src/components/profile-details/ProfileDetails.tsx

```diff
@@ -17,19 +17,12 @@ function ProfileDetails({ user }: ProfileDetailsProps) {
     setIsEditing(false);
   };
 
-  const updateUser = (updatedUser: User) => {
```

Comment:
Note that not every method the updates the store will always be in the store's file.
A lot of times the functions are specific to a specific context and then it makes sense to put them in the speicific component they are related to. 
(In this case it's OK to put them in the store, just making a general remark)